downstairs/src/region.rs

```diff
@@ -2129,7 +2131,7 @@ impl Region {
                 if el > self.def.extent_count().try_into().unwrap() {
                     crucible_bail!(InvalidExtent);
                 }
-                el
+                el + 1 // +1 because our loop is exclusive.
```

Comment:
Was this a bug before? Should this go in separately?

Reply:
It was there before, but the extent limit is not used unless you are doing LiveRepair.

It was more a "I don't know quite how this will work yet" than an actual bug :)